XcodeServerSDK/Utils/Logging.swift

```diff
@@ -50,7 +50,7 @@ public class ConsoleLogger: Logger {
     }
     
     public func log(message: String) {
-        println(message)
+        print(message)
```

Comment:
You have to be joking... :laughing: Way to make Swift less usable, wow. Ok, keep it this way then, I'll take a look once we merge it.